parastell/parastell.py

```diff
@@ -191,6 +191,27 @@ def export_invessel_build(
                 dagmc_filename=dagmc_filename, export_dir=export_dir
             )
 
+    def export_invessel_component_mesh(
```

Comment:
I think we might want to match the name of this method to that of the class for consistency, i.e., `export_invessel_build_mesh`.

Reply:
I would still keep 'component' in the method's name since it meshes specific components, defined as an argument, rather than the entire radial build. We could have another method for meshing the entire build, though.